### DISCUSSION TITLE
Unable to logout of chat GPT

### DISCUSSION THREAD

**Discussion by singh-jaskirat** [lencx/ChatGPT#1171] [Q&A]
![image](https://github.com/lencx/ChatGPT/assets/32123175/0de2c15e-9766-4e01-9035-11d4c8c8bcdd)

Hi All, My password changed and I am unable to see the profile and logout option. I have tried the following.
1. Clearing config
2. Delete .chatgpt folder in user profile
3. Uninstall and reinstall chatgpt

However; I am unable to logout or update my password. Can someone share how it can be done? Thanks.

**Comment by bronglil** [comment]
If you are using web just go to application and clear all session then do a hard refresh it will work for you.
